include/neml2/models/Model.h

```diff
@@ -44,27 +47,17 @@ namespace neml2
  * method \p set_value. All concrete models must provide the implementation of the forward operator
  * by overriding the \p set_value method.
  */
-class Model : public NEML2Object, public LabeledAxisInterface, public NonlinearSystem
+class ModelBase : public DataStore, public LabeledAxisInterface, public NonlinearSystem
```

Comment:
```suggestion
class Model : public ContainsParameters<Data>, public LabeledAxisInterface, public NonlinearSystem
```
so basically the same suggestion for `DataStore`.

Reply:
Same thing as before, this doesn't work.